src/core/mac/sub_mac.cpp

```diff
@@ -60,6 +60,8 @@ SubMac::SubMac(Instance &aInstance)
     , mPcapCallback(NULL)
     , mPcapCallbackContext(NULL)
     , mKeyId(0)
+    , mMacFrameCounter(0)
```

Comment:
Why change it this way. This member was moved from class `KeyManager` to `SubMac`. There are member called `mMleFrameCounter` and `mStoredMleFrameCounter`. The naming of `mMacFrameCounter` makes it easy for reader to understand which frame counter this member holds.

Reply:
This is a member of MAC layer itself. I don't think the `Mac` prefix is necessary. Otherwise, we'll need `mMacState`. Thoughts? The `mMleFrameCounter` in KeyManager, which doesn't belong to mac layer or MLE layer.

Reply:
Sound reasonable. Will change that. Thanks!